src/pages/App.tsx

```diff
@@ -76,85 +79,91 @@ export default function App(): ReactElement {
     <Suspense fallback={null}>
       <Web3ReactManager>
         <BasicPoolsProvider>
-          <MinichefProvider>
-            <TokensProvider>
-              <UserStateProvider>
-                <GasAndTokenPrices>
-                  <PendingSwapsProvider>
-                    <RewardsBalancesProvider>
-                      <AppContainer>
-                        <TopMenu />
-                        <Switch>
-                          <Route exact path="/" component={Swap} />
-                          <Route exact path="/pools" component={Pools} />
-                          {pools.map(({ name }) => (
-                            <Route
-                              exact
-                              path={`/pools/${name}/deposit`}
-                              render={(props) => (
-                                <Deposit {...props} poolName={name} />
-                              )}
-                              key={`${name}-name-deposit`}
-                            />
-                          ))}
-                          {pools.map(({ name, route }) => (
-                            <Route
-                              exact
-                              path={`/pools/${route}/deposit`}
-                              render={(props) => (
-                                <Deposit {...props} poolName={name} />
-                              )}
-                              key={`${route}-route-deposit`}
+          <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
Would you mind moving this below the various "Data" providers? Just above `AppContainer` should be fine

Reply:
Good question!
Since this `Provider` is using for `DatePicker`, so I think there will no problem if we move this above `AppContainer`!